paying_for_college/tests/test_scripts.py

```diff
@@ -35,15 +35,20 @@ class PurgeTests(django.test.TestCase):
     fixtures = ['test_fixture.json', 'test_program.json']
 
     def test_purges(self):
+        self.assertFalse(Program.objects.count() == 0)
```

Comment:
That's better, although in this case I think we'd need 
```python
self.assertTrue(Notification.objects.exists())
```
We want the prove Notifications exist before wiping them.